include.js

```diff
@@ -2037,6 +2037,13 @@ let cards = [
     imageLink: './Art/SamilAbud/bouncing-effect.gif',
     author: 'Samil Abud',
     githubLink: 'https://github.com/samilabud'
+  },
+  {
+    artName: 'HappyCat',
+    pageLink: './Art/dvasquez08/dave.html',
+    imageLink: './Art/dvasquez08/happy-cat.gif',
+    author: 'David Vasquez',
+    githubLink: 'https://github.com/dvasquez08'
```

Comment:
Where is the rest of the contribution? 🤔

Reply:
Oh I thought I added that...sorry about, I'm adding now